unittests/src/main/java/uk/ac/manchester/tornado/unittests/tools/TornadoHelper.java

```diff
@@ -153,6 +154,14 @@ static void runTestVerbose(String klassName, String methodName) throws ClassNotF
                 bufferFile.append(message);
                 successCounter++;
             } else {
+                // If UnsupportedConfigurationException is thrown this means that test did not fail, it simply can't be run on current configuration
```

Comment:
Can you pass the auto-formatter? 

https://github.com/beehive-lab/TornadoVM/blob/master/assembly/src/docs/3_INTELLIJ.md

Reply:
Done!